fixtures/js-stubs/replayRecord.ts

```diff
@@ -9,6 +9,8 @@ export function ReplayRecord(replayRecord: Partial<TReplayRecord> = {}): TReplay
       name: 'Other',
       version: '',
     },
+    count_dead_clicks: 1,
```

Comment:
this file gave me an error unless I added the two fields in

Reply:
👍 this needed to be updated because of the related type changes in static/app/views/replays/types.tsx